templates/launchd.plist

```diff
@@ -54,5 +54,12 @@
       <key>BUILDKITE_AGENT_CONFIG</key>
       <string>/Users/your-build-user/.buildkite-agent/buildkite-agent.cfg</string>
     </dict>
+    <key>LimitLoadToSessionType</key>
```

Comment:
For consistency can there be a blank line between the previous item and this new one?